src/stylesheets/core/_utilities.scss

```diff
@@ -26,22 +26,12 @@
 // Unstyled list helper
 
 @mixin unstyled-list() {
-  display: block;
+  @include margin(0 null);
```

Comment:
Right, I wasn't suggesting adding `-webkit-margin-before` ourselves. I meant should we add `margin-bottom: 1em;` ourselves, since webkit is already doing it + to make it more consistent across browsers. What do you think?